.github/workflows/tests.yaml

```diff
@@ -167,6 +171,10 @@ jobs:
                   # The idea is to not upload our objdir unless builds have
                   # actually succeeded, because that just wastes space.
                   rsync -a out/debug/standalone/ objdir-clone || true
+            - name: Build simulated device
+              timeout-minutes: 5
+              run: |
+                  scripts/examples/gn_build_test_example.sh app1
```

Comment:
Wait, actually, why are we building this here, where it's not used?  The jobs where we just build various things are `.github/workflows/examples-linux-standalone.yaml` and `.github/workflows/darwin.yaml`; the things we build in _this_ job should be things this job actually needs to run.

Reply:
Ok. Thank you.

Reply:
I am going to remove this change, because the files are generated in this script which we will need to install the node modules. I will create another PR to add the generated files to zzz_generated folder.